app/modules/asset_groups/models.py

```diff
@@ -461,19 +461,22 @@ def detected(self, job_id, response):
 
             for annot_id in range(len(results)):
                 annot_data = results[annot_id]
-                annot_uuid = annot_data.get('uuid', {}).get('__UUID__')
+                content_guid = annot_data.get('uuid', {}).get('__UUID__')
                 ia_class = annot_data.get('class', None)
-                if not annot_uuid or not ia_class:
+                viewpoint = annot_data.get('viewpoint', None)
+                if not viewpoint or not ia_class:
                     raise HoustonException(
-                        log, 'Need a uuid and a class in each of the results'
+                        log, 'Need a viewpoint and a class in each of the results'
```

Comment:
This is an I don't know question buut does this mean that the content guid is optioonal? If so how/why?

Reply:
yes - or so i claim.  as to why; well, i am just basing this on the [analog in Asset model](https://github.com/WildMeOrg/houston/blob/main/app/modules/assets/models.py#L44).  in the annot case, when it is created from detection, we _should_ always have this value to set.  when we (eventually) manually create annots in the future, we would have to compute it.

perhaps the real question (outside the scope of this pr?) is: what is it used for?  in wbia, it is my understanding it is greatly important to know when you are actually talking about "the same annotation" from a visual/pixel/content point of view.  do we ever care about that in houston or from a user standpoint?  not really right now anyway.   so i think optional is fine.  (it seems like we currently _do not_ set the one on asset at this point, fwiw.)